SpatialGDK/Source/SpatialGDK/Public/Utils/SchemaUtils.h

```diff
@@ -122,6 +122,10 @@ inline void AddObjectRefToSchema(Schema_Object* Object, Schema_FieldId Id, const
 	{
 		AddObjectRefToSchema(ObjectRefObject, 5, *ObjectRef.Outer);
 	}
+	if (ObjectRef.bSingletonRef)
+	{
+		Schema_AddBool(ObjectRefObject, 6, ObjectRef.bSingletonRef);
```

Comment:
Is there a plan to eventually add constants for the values in ObjectRef schema?

Reply:
I'll add them in this PR.